spec/requests/api/v1/registrations_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Api::V1::Registrations', type: :request do
+  let(:email)                 { Faker::Internet.unique.email }
+  let(:password)              { Faker::Internet.password(min_length: 8) }
+  let(:gender)                { Faker::Gender.binary_type }
+
+  let(:params) do
+    { email: email, gender: gender, password: password, password_confirmation: password }
+  end
+
+  describe 'POST /create' do
+    context 'with valid parameters' do
+      it 'creates a new User' do
+        expect do
+          post '/api/v1', params: params
+        end.to change(User, :count).by(1)
+      end
+
+      it 'successful response' do
+        post '/api/v1', params: params, as: :json
```

Comment:
You can use `subject`.

See https://relishapp.com/rspec/rspec-core/docs/subject/explicit-subject